doc/tools/parse_output_variables.py

```diff
@@ -237,21 +237,22 @@ def main():
                 arguments = get_arguments_based_on_comma_locations(working_line, comma_locations)
 
                 # parse out data from the variable name first
-                variable_name, variable_units = process_variable_name_and_units(arguments[0])
+                variable_name = arguments[1]
```

Comment:
Fixed up the SetupOutputVariable parser to account for the "new" state argument and also the "new" units argument.